com.unity.shadergraph/Editor/GraphUI/DataModel/Constants/ICLDSConstant.cs

```diff
@@ -0,0 +1,65 @@
+
+using System;
+using UnityEditor.GraphToolsFoundation.Overdrive;
+using UnityEditor.ShaderGraph.GraphDelta;
+using UnityEngine;
+using UnityEngine.GraphToolsFoundation.Overdrive;
+
+namespace UnityEditor.ShaderGraph.GraphUI
+{
+    [Serializable]
+    public abstract class ICLDSConstant : IConstant, ISerializationCallbackReceiver
```

Comment:
Why `ICLDSConstant` rather than `CLDSConstant` ?

Reply:
Oh, it was an interface at one point-- I'll change the name. Really it should be something like BaseShaderGraphConstant.